src/nifake/metadata/functions.py

```diff
@@ -1248,15 +1248,49 @@
             'description': 'This function takes one parameter other than the session, which happens to be an enum and has a default value defined in functions_addon.',
         },
     },
+    'ArrayInputFunction': {
+        'codegen_method': 'public',
+        'returns': 'ViStatus',
+        'parameters': [
+            {
+                'direction': 'in',
+                'enum': None,
+                'name': 'vi',
+                'type': 'ViSession',
+                'documentation': {
+                    'description': 'Identifies a particular instrument session. You obtain the **vi**',
+                },
+            },
+            {
+                'direction': 'in',
+                'enum': None,
+                'name': 'bufferSize',
```

Comment:
rename to size or numberOfElements
(buffer size is in bytes usually)

Reply:
fixed